salt/orch/update.sls

```diff
@@ -555,3 +560,29 @@ remove-update-grain:
         destructive: True
     - require:
       - remove-caasp-fqdn-grain
+
+{% if is_migration %}
+reenable-transactional-update-timer:
+  salt.function:
+    - tgt: '{{ migrated_nodes }}'
+    - tgt_type: compound
+    - batch: 3
+    - name: service.enable
+    - arg:
+        - transactional-update.timer
+
+unset-migration-grains:
+  salt.function:
+    - tgt: '{{ migrated_nodes }}'
+    - tgt_type: compound
+    - name: grains.delval
+    - arg:
+        - tx_update_migration_available
+        - tx_update_migration_notes
+        - tx_update_migration_mirror_synced
+        - migration_in_progress
```

Comment:
I don't see where all these grains are set...

Reply:
they're eventually set by this service: https://github.com/thkukuk/update-checker